Dockerfile

```diff
@@ -23,9 +23,11 @@ ENV MIX_ENV=prod
 
 WORKDIR /opt/app
 
-COPY config config
-COPY mix.* ./
-RUN mix do deps.get --only=$MIX_ENV, deps.compile
+COPY mix.exs mix.lock ./
```

Comment:
The changes here are mostly due to how Docker caches. Each line creates a new layer. Based on what needs to be recompiled, separating some steps and copying at different points helps maximize the Docker cache, which speeds up the build. 

These are picked up from the new Phoenix default Dockerfile.